app/konnect/analytics/use-cases.md

```diff
@@ -0,0 +1,185 @@
+* Monetization: If you have a monetization strategy, knowing which applications are actually consuming the data can help you allocate resources to make financial decisions. 
+* Innovation: With these reports you can identify API usage patterns and trends, which can help you allocate resources and make development decisions. 
+
+### Latency for an API over the last 30 days
+
+At this point, you have multiple reports that display how your company's APIs are performing generally.
+
+Your company determines that it is critical that payments are processed quickly using the Payments API, so you decide to generate a report that will tell you how performant the Payment API is over the last 30 days. To build the report, set the following options in the UI:
+
+* **Name**: Payment API - Latency (last 30 days)
+* **Date/Time**: Last 30 days
+* **Chart type**: Line chart
+* **Select a metric**: Response latency (p99)
+* **Group by**: Service
+* **Choose granularity**: Daily
+* **Entity Selection**: Payment 
+
+Add a filter for the Payment API. Click on **Add Filter**, then set the following options:
+ 
+* **Filter by**: Service
+* **Operator**: In
+* **Filter value**: Payment
+
+[screenshot of finished report]
+
+This report can provide your stakeholders with the answers to questions like:
+
+* How performant has my API been over the last 30 days?
+* Is my Payment API processing payments quickly enough?
+* Are there any performance outliers?
+
+## Share reports
+
+Now that you have your reports ready, you might want to share them with your stakeholders. 
+
+You can do this in a few different ways:
+
+* Grant your stakeholders the Analytics Viewer [role](/konnect/org-management/teams-and-roles/teams-reference) in 
+the {{site.konnect_short_name}} organization to access reports in a read-only state, 
+or the Analytics Editor role to let them customize the reports for themselves.
+* Take screenshots of the reports and share the images.
+* Export any of the reports you created into CSV files. This gives you raw data that you can 
+plug into any of your external dashboards or visualization services.
+
+    Open one of the reports you created, then from the **Reports Actions** dropdown, select **Export Report as CSV**.
+
+
+## Conclusion 
+
+After completing these example scenarios, you should have a better understanding of how you can use a variety of metrics to collect analytics on KPIs with {{site.konnect_short_name}} custom reports. 
+
+Additionally, you can use these reports to find and solve problems in resource allocation, optimization, and performance. You now know how to use the information gathered from those reports, such as anomalies, to further drill down into what is causing the problem. This information can then be presented to stakeholders to ensure your APIs are secure, performant, and stable. 
```

Comment:
I suggest taking a more narrative tone to keep it consistent with the rest of the page. Something like:

> Now that you've created these reports and gleaned some insights from the data, you can be more informed when you talk to your stakeholders. 
>
> You learned that:
> * One of the APIs was {doing something}
> * {During some date range, this other thing was happening}
> * Users are focusing on {X applications}, which means that {something}
> * Something else maybe
>
> You can share the reports and dig into any anomalies, performance, or stability issues, or let them draw conclusions based on further analysis.

Reply:
@lena-larionova I like your ideas about changing the tone and "You can share the reports and dig into any anomalies, performance, or stability issues, or let stakeholders draw conclusions based on further analysis."

For the "You learned" bullet points, I think those would be good for a regular tutorial, but since the outcome of this tutorial is more "you learned how to think about custom reports so you can now go out and make your own" instead of  "here's these specific reports that you created", I wonder if those should be phrased differently? I like that idea for a list though. Maybe something like:

> Now that you've created these reports and gleaned some insights from the data, you can be more informed when you talk to your stakeholders.

>  You learned that:

> - Custom reports can be used to view metrics like number of requests, latency, and throughput
> - You can use information from reports to dig deeper into any anomalies, performance, or stability issues to determine what is causing the problem
> - After you create a report, you can export and share the reports with stakeholders to let them draw conclusions based on further analysis

> If you want to continue learning about ways to use {{site.konnect_short_name}} to analyze data, see [Analyze Services and Routes](/konnect/analytics/services-and-routes/). 

Not sure about that last doc link, but something like that that would help them continue learning or move on to something else that's related.

Reply:
Love this - let's go with it.